### PR TITLE
Clarify an answer and fix some typos

### PR DESCRIPTION
**Pull request by patridge**
Reading through the OS distribution chart, I found myself conflating OS version and app version. I tried to reword it to be more clear based on my own understanding. Feel free to adjust it further if I have misinterpreted that answer.

I also fixed a couple random typos I noticed while reviewing the rest of the FAQ.

### CODE REVIEW

**Comment by PRMerger8** [comment]
@patridge : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger20** [comment]
@King-of-Spades : Thanks for your contribution! The author(s) have been notified to review your proposed change. @blparr

**Review by King-of-Spades — APPROVED** [review]
